pkg/config/config.go

```diff
@@ -95,6 +96,7 @@ var defaultServerConfig = &ServerConfig{
 			MaxExpiresIn: config.Duration{Duration: time.Hour},
 		},
 	},
+	ReadHeaderTimeoutSeconds: 32,
```

Comment:
mind adding a similar comment as kubernetes does for why this value?

Reply:
updated it